rfcs/0000-template.md

```diff
@@ -0,0 +1,67 @@
+---
+Start Date: DD-MM-YYYY
+RFC PR: (leave this empty if no PR yet)
+Blade Issue: (leave this empty if no issue yet)
+---
+
+# Title of the RFC <!-- omit in toc -->
+
+### Table Of Contents <!-- omit in toc -->
+- [Summary](#summary)
+- [Basic Example](#basic-example)
+- [Motivation](#motivation)
+- [Detailed Design](#detailed-design)
+- [Drawbacks/Constraints](#drawbacksconstraints)
+- [Alternatives](#alternatives)
+- [Adoption strategy](#adoption-strategy)
+- [How do we educate people?](#how-do-we-educate-people)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+# Summary
+Brief explanation about the proposal.
+
+# Basic Example
+Include a basic code example. Omit this section if it's not applicable.
+
+# Motivation
+- Why are we doing this?
+- What use cases does it support?
+- What is the expected outcome?
+
+Try to focus on explaining the motivation so that if this RFC is not accepted, the motivation could be used to develop alternative solutions. In other words, try to list down the constraints you are trying to solve without coupling them too closely to the solution you have in mind.
+
+# Detailed Design
+This is the bulk of the RFC. Explain the design in enough detail for somebody familiar with the Design System to understand, and for somebody familiar with the implementation to implement. This should get into specifics and corner-cases, and include examples of how the feature is used. Any new terminology should be defined here.
+
+# Drawbacks/Constraints
+Why should we *not* do this? Maybe try to consider the following constraints
+- Implementation cost, both in terms of code size and complexity.
```

Comment:
what does code size mean? Man hours/Time required?

Reply:
Not sure if the RFC authors would be able to give man hours at time of RFC.
Probably, we can provide a time range based size. Like:
- Small (0-5 days or 1 week)
- Medium (5-10 days or 2 weeks)
- Large (More than 10 days or more than 2 weeks)
- XLarge (More than 20 days or more than 4 weeks)

Reply:
This doesn't asks you to give time estimates. It just tells you to list down constraints taking these perspectives in mind